### ISSUE TITLE
Pretraining dataset details

### ISSUE THREAD

**Issue by Hpsoyl** [computational-cell-analytics/micro-sam#994]
Thank you for your awesome work! When following the provided Jupyter notebook for fine-tuning, the model (vit_b.pt) is downloaded from:
https://uk1s3.embassy.ebi.ac.uk/public-datasets/bioimage.io/diplomatic-bug/1.1/files/vit_b.pt. I have two specific questions: Is this vit_b.pt file V1, V2, or V3 of the model? If it is V3, were the test or validation splits of the Dynamic Nuclear Net dataset (from Schwartz et al.) used during pretraining?

**Comment by Hpsoyl** [comment]
Thank you very much, I've got this clearly!

**Comment by Hpsoyl** [comment]
@anwai98 Hello, we encountered some issues while testing your code. We found that the mSA score obtained using the official v3 weights on the Dynamic Nuclear Net dataset is higher than what was reported in the paper. Our testing script is as follows:
```
from glob import glob
import os
os.environ["CUDA_VISIBLE_DEVICES"] = "5" 

from micro_sam.evaluation import inference
from micro_sam.evaluation.evaluation import run_evaluation
from micro_sam.training import util

def get_paths(n_val_images=2, split=None):
    image_paths = sorted(glob(os.path.join(image_dir, "*")))
    gt_paths = sorted(glob(os.path.join(segmentation_dir, "*")))
    
    if split is None:
        return image_paths, gt_paths

    else:
        if split == "val":
            return image_paths[:n_val_images], gt_paths[:n_val_images]
        elif split == "test":
            return image_paths[n_val_images:], gt_paths[n_val_images:]
        else:
            raise ValueError(f"'{split}' is not a valid split name.")

root_dir = "output"
image_dir = "/data2/syhong/DynamicNuclearNet-Seg/test/X/"
segmentation_dir = "/data2/syhong/DynamicNuclearNet-Seg/test/y/"

# We define the paths to the validation inputs and the test inputs# We will use these path variables throughout the scripts
val_image_paths, val_gt_paths = get_paths(split="val")
test_image_paths, test_gt_paths = get_paths(split="test")

checkpoint=None  # overwrite in case you have custom checkpoints to load models from
model_type = "vit_b_lm"  # overwrite with your desired choice of model
experiment_folder = os.path.join(root_dir, "experiment_dir", "DynamicNet", "vit_b_lm")

prediction_folder = inference.run_instance_segmentation_with_decoder(
    checkpoint=checkpoint,
    model_type=model_type,
    experiment_folder=experiment_folder,
    val_image_paths=val_image_paths,
    val_gt_paths=val_gt_paths,
    test_image_paths=test_image_paths,
)

print("Evaluating", prediction_folder)
prediction_paths = sorted(glob(os.path.join(prediction_folder, "*")))
save_path = os.path.join(experiment_folder, "results", "instance_segmentation_with_decoder.csv")
results = run_evaluation(
    gt_paths=test_gt_paths,
    prediction_paths=prediction_paths,
    save_path=save_path
)
print(results)
```
When evaluating on the Dynamic Net test set, we obtained an mSA score of 0.58805, which appears inconsistent with the results in Extended Data Fig. 2(b). To verify the correctness of our testing script, could you kindly provide the exact values from the original test results? You may reach me at: [hongshiyao@mail.ustc.edu.cn]
Thank you for your assistance and for your impressive work!

**Comment by constantinpape** [comment]
> We found that the mSA score obtained using the official v3 weights on the Dynamic Nuclear Net dataset is higher than what was reported in the paper.

The numbers in the paper were produced with the v2 weights. The v2 model was not trained on the DynamicNuclearNet train split whereas the v3 weights are trained on this train split. So what you report here is expected.

If you want to reproduce the numbers from the extended data figure you would need to use the v2 weights.
See https://computational-cell-analytics.github.io/micro-sam/micro_sam.html#other-models for how to obtain them.
